client/src/utils/utils.ts

```diff
@@ -40,3 +40,8 @@ export const metadataToProject = (
 
   return p;
 };
+
+export const getProjectNumberFromId = (id: string): number => {
+  const split = id.split(":");
+  return parseInt(split[split.length - 1], 10);
```

Comment:
I know the final result is the same but I would prefer splitting and returning the 3 parts and throwing an exception if they are not 3. wdyt?

Reply:
That works, I will change the function to return all bits - it is helpful in some places too